docker-compose.yml

```diff
@@ -11,7 +10,17 @@ services:
     volumes:
     - ./psql_data:/var/lib/postgresql/data
     - ./.docker/initdb.sql:/docker-entrypoint-initdb.d/initdb.sql
```

Comment:
should we then remove this initdb.sql link?

Reply:
and possibly move the file to the 'ci' where it afaik is now the only place it is used?